src/Amazon.AspNetCore.Identity.Cognito/CognitoUserClaimsPrincipalFactory.cs

```diff
@@ -59,5 +61,44 @@ public async Task<ClaimsPrincipal> CreateAsync(TUser user)
             var claimsIdentity = new ClaimsIdentity(claims, IdentityConstants.ApplicationScheme);
             return new ClaimsPrincipal(claimsIdentity);
         }
+
+        /// <summary>
+        /// Internal method to map System.Security.Claims.ClaimTypes to Cognito Standard Attributes
+        /// </summary>
+        /// <param name="claims"></param>
+        private void MapClaimTypesToCognito(List<Claim> claims)
```

Comment:
IEnumerable parameter

Reply:
Done

Reply:
I need it to be a List to access the .Add() method.

I think this is OK as it is a private utility method.